jobs/build/promote/Jenkinsfile

```diff
@@ -228,15 +230,19 @@ node {
         if (!params.VERSION) {
             error("You must explicitly specify a VERSION to promote a release for a non-stream assembly.")
         }
+        if (params.ADVISORY) {
+            error("Cannot override image advisory number for a non-stream assembly.")
+        }
         (major, minor) = commonlib.extractMajorMinorVersionNumbers(params.VERSION)
         group = "openshift-${major}.${minor}"
         // FIXME: Only support standard X.Y.Z and hotfix releases for now. RCs and FCs are not supported yet.
```

Comment:
aren't they?

Reply:
I don't understand your meaning here.
This check makes sure advisory numbers are all defined in releases.yml. I don't think there's any use cases to specify image advisory using build parameter.